File: app/src/main/java/com/bmn/bookfinder/data/network/remote/ApiUtils.java
package com.bmn.bookfinder.data.network.remote;

import android.content.Context;

import com.bmn.bookfinder.helpers.Constants;

public class ApiUtils {
    public static final String QUERY_SUBJECT = "subject:%s";

    public static ApiCalls getGoogleApiService(Context context) {
        return ApiClient.getApi(context, Constants.Google.BASE_URL).create(ApiCalls.class);
    }
}

File: app/src/main/java/com/bmn/bookfinder/data/network/remote/ApiUtils.kt
package com.bmn.bookfinder.data.network.remote

import android.content.Context
import com.bmn.bookfinder.helpers.Google

object ApiUtils {
    const val QUERY_SUBJECT = "subject:%s"
    @JvmStatic
    fun getGoogleApiService(context: Context?): ApiCalls {
        return ApiClient.getApi(context, Google.BASE_URL).create(ApiCalls::class.java)
    }
}